js/private/coverage/BUILD.bazel

```diff
@@ -9,7 +9,7 @@ exports_files([
 write_source_files(
     name = "coverage_checked",
     files = {
-        "coverage.js": "//js/private/coverage/bundle:bundle.js",
+        "@@//js/private/coverage:coverage.js": "@@//js/private/coverage/bundle:bundle.js",
```

Comment:
oh, does this mean it could work with bazel-lib before the hacky workaround (v1.13.0) ?

Reply:
still investigating

Reply:
popped this out of DRAFT before I noticed the issue. will put in back for now